app/code/community/Bolt/Boltpay/etc/config.xml

```diff
@@ -126,8 +134,34 @@
           </deferred>
         </states>
       </order>
+      <!-- Adds total sorting refinement due to the changes in uasort between PHP5 and PHP7 -->
```

Comment:
is this part of this on-hold change? If not, can you create a separate PR (I can approve instantly).

Reply:
In PHP7, for Magento 1.9.x, the grand total will always be wrong in the store backend without this being set, so, for me, this is part of the same pull request.  FYI, nnly in Wordpress/WooCommerce, for total code structure change have I placed several unrelated code changes into the same pull request.  Outside of this, I always have and always will separate unrelated code changes as separate PR as my standard practice.